media-libs/aribb24/aribb24-1.0.3-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="Library for decoding ARIB STD-B24 subtitles"
+HOMEPAGE="https://github.com/nkoriyama/aribb24"
+SRC_URI="https://github.com/nkoriyama/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+
+RDEPEND="media-libs/libpng:0=[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
```

Comment:
With EAPI-7, this is a BDEPEND instead. Looks good otherwise!

Reply:
Fixed, thanks!